src/SerializableAsync.ts

```diff
@@ -35,9 +35,11 @@ export class SerializableAsync extends SerializableBase implements ISerializable
         return await this.fromUnknown(obj)
     }
 
-    public static async deserialize(value: string, type?: new () => SerializableAsync): Promise<SerializableAsync> {
-        if (type) {
-            return await this.deserializeT(value, type)
+    public static async deserialize(value: string): Promise<SerializableAsync> {
```

Comment:
Sollte die Implementierung dieser Methode nicht mehr wie die in `Serializable` aussehen? Aktuell unterscheiden sie sich nämlich sehr stark.